### DISCUSSION TITLE
Proposal on financial & operational transparency of Protocol Labs and Filecoin Foundation

### DISCUSSION THREAD

**Discussion by sxxfuture-official** [filecoin-project/FIPs#943] [Enhancements - Governance]
# Proposer

SXX Future Data from Asia Storage Provider Working Group

# Background

The Filecoin network has reached a critical moment in the blockchain industry. To rekindle the enthusiasm of the network participants and unite consensus of the community members, the community needs greater financial & operational transparency from Protocol Labs (PL) and Filecoin Foundation (FF).

At the moment, the number of vesting tokens of PL and FF have greatly exceeded the total amount of tokens mined by the Filecoin Storage Providers(SPs) community. As the FIL increases, so does the community require a transparent financial planning to bring confidence of priority and token holders’ expectations. Brining community members’ participation in the governance, operational priorities, financial plannings and financial audit could be helpful to enhance the correct usage of the network resources and to rally community confidence and rally consensus.

# Proposal

1. **Clarify Roles and Responsibilities of PL and FF:**  Filecoin Foundation’s principles as a blockchain project foundation. Protocol Labs as core developers of the Filecoin network. The foundation even though registered as a commercial entity in US, should behave and define its status like an non-profit organization or a software foundation, and its main responsibility is to secure the network’s fund for the future, help community to govern the network changes (not letting core members control this process) and the ecosystem prosperity of the Filecoin network. The long-term goal of the foundation is to take the vested FIL tokens and independently support the work of any team developing on Filecoin network. Protocol Labs need to use its vested FIL tokens to continue to fuel Filecoin protocol’s core development, making it more secure and performant and fund Web3.0 related protocol development. In the long run, the foundation and the lab should not be the only contributor and leader of Filecoin, but a part of the entire ecosystem of Filecoin.
2. **Clarify the Liquidation Process of PL and FF:**  PL and FF should publicly disclose the vested token liquidation process, which their daily operations depend on. The excess tokens or assets should be locked or restricted until the next financial year's financial planning is approved by the community in which a new liquidation quota is granted.
3. **Transparent Financial and Operational Planning:**  PL and FF Foundation should develop financial plans for each financial year and submit it to the community for discussion and voting for approval. Publish financial statements every six to twelve months, and subject to audits by the community or third party.  Audit reports should be disclosed regularly.
4. **Disclosure of Current Token Holdings:**  Fully disclose the address and quantity of all token holdings. Future liquidation plans need to conform to the financial and plannings. There needs to be a planned liquidation amount (per annum and per month) and detailed plans for management of vesting balance (disclose vested address and quantity). After the liquidation plan is approved by the community, a financial manager is to be appointed, who is responsible for reporting the numbers of token that has been liquidated, to be liquidated and liquidation quota on a monthly basis;

	We strongly appeal that Protocol Labs and Filecoin Foundation should work together to share transparency in their respective FIL token, formulate a clear liquidation process, allow the community to participate/approve plans  and accept audits. As one of the largest single holders of FIL tokens, Protocol Labs should shoulder its responsibilities, show its long-term planning for Filecoin network, and long-term commitment in Filecoin community, and promote the sustainable development of web3 storage network together.

**Comment by jaychoucc** [comment]
support

**Reply by Yuehuajin1991** [reply]
同意 。让FIL再次伟大

**Reply by 205312327** [reply]
同意 。让FIL再次伟大

**Comment by Ming-Filaccount** [comment]
very good! Filcoin keep going！

**Reply by Yuehuajin1991** [reply]
同意✅

**Reply by nddiandao** [reply]
支持

**Comment by kkkkkkww** [comment]
同意 二娃子叫我來的 讓FIL 再次偉大!!!

**Reply by JIANGRUILIN** [reply]
二娃子：我知道是谁了

**Comment by ggyy2024** [comment]
support

**Reply by 205312327** [reply]
支持

**Comment by fil88888888** [comment]
同意

**Comment by fil88888888** [comment]
支持

**Comment by kongxinputi** [comment]
support

**Reply by 205312327** [reply]
支持

**Comment by qtdslly** [comment]
支持

**Reply by toutuoge** [reply]
支持

**Comment by toutuoge** [comment]
支持

**Reply by Ch248248** [reply]
让Fil变的越来越好

**Reply by Ch248248** [reply]
支持

**Reply by 205312327** [reply]
让Fil生态越来越强，蓬勃发展

**Comment by jingyu0212** [comment]
强烈支持！👍

**Reply by 205312327** [reply]
强烈支持

**Comment by HJann762** [comment]
支持

**Comment by Bllaaa** [comment]
强烈支持

**Comment by Derek-zd** [comment]
Big 胆！！！

**Comment by Fatman13** [comment]
> The foundation even though registered as a commercial entity in US, should behave and define its status like an **non-profit organization**

顶！

**Comment by JIANGRUILIN** [comment]
支持，这样有利于项目的发展和社区的自理！

**Reply by 205312327** [reply]
支持，这样有利于项目的发展和社区的自理！

**Comment by plutoyi** [comment]
support

**Comment by small-masterL** [comment]
support

**Comment by sharkfee** [comment]
同意

**Comment by a351765899** [comment]
support

**Comment by cj091116** [comment]
👍support

**Comment by xiaomdong** [comment]
support

**Comment by AlanGreaterheat** [comment]
support !!!

**Comment by a351765899** [comment]
support !!!

**Reply by nddiandao** [reply]
support ！

**Comment by fil88888888** [comment]
支持



---Original---
From: ***@***.***&gt;
Date: Mon, Feb 26, 2024 12:28 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [filecoin-project/FIPs] Proposal on financial &amp; operationaltransparency of Protocol Labs and Filecoin Foundation (Discussion #943)




 
support !!!

**Comment by fil88888888** [comment]
支持



---Original---
From: ***@***.***&gt;
Date: Sun, Feb 25, 2024 23:23 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [filecoin-project/FIPs] Proposal on financial &amp; operationaltransparency of Protocol Labs and Filecoin Foundation (Discussion #943)




 
support !!!

**Comment by jordanlynnf** [comment]
Hello, everyone! We’re excited to share the FF 2023 Annual Report (https://fil.org/blog/filecoin-foundation-2023-annual-report/) is live. The report highlights our efforts and collaborations over the past year and serves as a touchpoint to reflect on the potential of the Filecoin network.

**Reply by ansermino** [reply]
Thanks for sharing. The link to the full report appears to be dead.

**Comment by JIANGRUILIN** [comment]
支持

**Reply by 205312327** [reply]
支持

**Comment by JIANGRUILIN** [comment]
支持！

**Reply by 205312327** [reply]
支持

**Comment by gugege521** [comment]
支持提案，希望fil能够更加公开透明，让社区力量真正能够发挥出来带动币价的起飞，更希望fil团队可以加快技术的研发，实现真正的技术突破。

**Reply by piaoliuping1** [reply]
同意
